BeautifulArrangements.java

```diff
@@ -0,0 +1,124 @@
+// Time Complexity :  O(k) [k: number of valid permutations]
```

Comment:
Check you Tc once. what are valid permutations? How many valid permutations can you have here?